client/lobbies/lobby.tsx

```diff
@@ -394,7 +155,55 @@ const Countdown = styled.div`
   margin: 16px 0;
 `
 
-export default class Lobby extends React.Component {
+function renderChatMessages(msg: Message) {
```

Comment:
similar to the comment on the last iteration of this, this probably makes more sense as `renderChatMessage`